src/core/ext/filters/client_channel/lb_policy/xds/xds_override_host.h

```diff
@@ -49,12 +49,16 @@ class XdsOverrideHostLbConfig : public LoadBalancingPolicy::Config {
     return child_config_;
   }
 
+  int override_host_status_mask() const { return override_host_status_mask_; }
```

Comment:
Instead of exposing this as an integer, how about adding an interface to provide set semantics?  That would make the calling code (both in the LB policy and in the config tests) more readable.

I'm thinking that we could add something like this in src/core/ext/xds/xds_health_status.h:

```
class XdsHealthStatusSet {
 public:
  explicit XdsHealthStatusSet(const std::vector<XdsHealthStatus>& statuses) {
    for (XdsHealthStatus status : statuses) {
      Add(status);
    }
  }

  bool operator==(const XdsHealthStatusSet& other) { return status_mask_ == other.status_mask_; }

  void Clear() { status_mask_ = 0; }

  void Add(XdsHealthStatus status) {
    status_mask_ |= (0x1 << status.status());
  }

  bool Contains(XdsHealthStatus status) {
    return status_mask_ & (0x1 << status.status());
  }

 private:
  int status_mask_ = 0;
};
```

Then we could have this method return `XdsHealthStatusSet`, and the callers could check whether the set contains the expected statuses.